server/src/api/user.rs

```diff
@@ -557,6 +558,11 @@ impl Perform for Oper<SaveUserSettings> {
       None => read_user.email,
     };
 
+    let bio = match &data.bio {
+      Some(bio) => Some(bio.to_owned()),
+      None => read_user.bio,
+    };
```

Comment:
This back-end will also need to check for too many characters in a bio, and throw an APIError if it is. That error string will have to be added to `en.json`.